config.yml

```diff
@@ -51,6 +52,30 @@ sites:
     # hosts:
       # - pmc.test
 
+  # https://github.com/wpcomvip/pmc-sportico
+  # https://bitbucket.org/penskemediacorp/pmc-sportico-2020/
+  pmc-sportico-2020:
+    skip_provisioning: false
+    description: "pmc-sportico"
+    repo: https://github.com/Varying-Vagrant-Vagrants/custom-site-template.git
+    custom:
+      delete_default_plugins: true
+      install_test_content: true
+      wpconfig_constants:
+        AUTOMATIC_UPDATER_DISABLED: true
+        DISALLOW_FILE_EDIT: true
+        DISALLOW_FILE_MODS: true
+        WP_DEBUG: false
+        WP_DEBUG_LOG: false
+        WP_DEBUG_DISPLAY: false
+        WP_DISABLE_FATAL_ERROR_HANDLER: false
```

Comment:
We'll need these constants as well:

```
PMC_IS_VIP_GO_SITE: true
VIP_GO_APP_ENVIRONMENT: true
VIP_GO_ENV: true
```

`PMC_IS_VIP_GO_SITE` is set based on `VIP_GO_ENV`, but there's no harm in being explicit in VVV.

Reply:
added! Thanks!